releasenotes/notes/add-primitives-support-31af39549b5e66e3.yaml

```
@@ -0,0 +1,31 @@
+---
+prelude: >
+    Qiskit Optimization 0.5 supports the new algorithms introduced in Qiskit Terra 0.22
+    which in turn rely on the `Qiskit Primitives <https://qiskit.org/documentation/apidoc/primitives.html>`_.
+    Qiskit Optimization 0.5 still supports the conventional algorithms based on ``QuantumInstance``.
+    But they will be deprecated in the future release.
```

```suggestion
    Qiskit Optimization 0.5 still supports the former algorithms based on ``QuantumInstance``,
    but they will be deprecated and then removed, along with the support here, in future releases.
```
I switched `conventional` to `former` to more indicate it was the older pre-existing algos. And the last bit I changed a little to add that these will be removed too along with the support code here in optimization.

---

Thanks. It's more sophisticated.